src/mapper/CommentMapper.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * CommentMapper File Doc Comment
+ *
+ * PHP Version 8.1.10
+ *
+ * @category Mapper
+ * @package  App\mapper
+ * @author   Marine Sanson <marine_sanson@yahoo.fr>
+ * @license  https://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     https://www.blog.marinesanson.fr/ Not inline for the moment
+ */
+declare(strict_types=1);
+
+namespace App\mapper;
+
+use App\entity\CommentEntity;
+use App\entity\PostEntity;
+use App\mapper\DateTimeMapper;
+use App\model\CommentModel;
+use App\model\PostDetailsModel;
+
+
+/**
+ * CommentMapper Class Doc Comment
+ *
+ * @category Mapper
+ * @package  App\mapper
+ * @author   Marine Sanson <marine_sanson@yahoo.fr>
+ * @license  https://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     https://www.blog.marinesanson.fr/ Not inline for the moment
+ */
+class CommentMapper
+{
+
+    /**
+     * Summary of _instance
+     *
+     * @var CommentMapper
+     */
+    private static $instance;
```

Comment:
```suggestion
    private static $_instance;
```
Pour rester cohérent

Reply:
Je l'avais fait mais j'ai eu une issue codacy pour quelques variables dont celles ci...